pkg/contractreader/extended_test.go

```diff
@@ -1,22 +1,22 @@
-package contractreader
+package contractreader_test
```

Comment:
:sweat_smile:

Reply:
This is because of mocking, I need to have a separate package to avoid cyclic dependency.